pkgs/top-level/all-packages.nix

```diff
@@ -33956,6 +33956,8 @@ with pkgs;
     gtk = gtk2;
   };
 
+  xborders = callPackage ../applications/window-managers/xborders { };
```

Comment:
This is not a window manager. Throw it under `tools/X11`.

Reply:
Alright. I noticed some things that were window manager-tangential such as picom were in `applications/window-managers`, so I thought it'd be appropriate to put it there since it's related. I'll move it.